### PR TITLE
Chronos: Delete existing tags and replace with new Installation options

### PR DESCRIPTION
**Pull request by liangs6212**
## Description

1. Delete existing tags
2. Replace the original tags with the new installation options.

### 1. Why the change?

Compatible with Installation options.

### 2. User API changes

No changed.

### 3. Summary of the change 

1. remove skip onnxrt16 and `tensorflow`

### 4. How to test?
- [ ] Unit test

### 5. pytest.mark Usage

pytest -m "mark1 and mark2" <your_path>/file.py

### CODE REVIEW

**Comment by liangs6212** [comment]
http://10.112.231.51:18889/view/BigDL-PR-Validation/job/BigDL-Chronos-PR-Validation/600/

File: python/chronos/test/bigdl/chronos/__init__.py [hunk 1]
```diff
@@ -13,3 +13,17 @@
...
+op_torch = pytest.mark.torch
+op_tf2 = pytest.mark.tf2
+
+# distribution and automl
+op_automl = pytest.mark.automl
+op_distributed = pytest.mark.distributed
+
+# other mark
+op_tf1 = pytest.mark.tf1
+op_all = pytest.mark.all
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/__init__.py:28]: maybe `op_all` can be replaced by `tag1 or tag2 ...` or we may need to give all ut a `op_all` tag?

**Review by TheaperDeng — COMMENTED** [review]
LGTM generally, will have an offline discussion later

File: python/chronos/test/bigdl/chronos/__init__.py (hunk 1 above)
Comment (liangs6212) [reply python/chronos/test/bigdl/chronos/__init__.py:28]: If we give all UTs op_all tags, then we can move the torch to the ut method, sounds good.
Usage: 
* pytest -m "all"  <your_path>/file.py
* pytest -m "torch and distributed" <your_path>/file.py
* pytest -m "torch and automl" <your_path>/file.py
* pytest -m "torch and distributed and not onnxrt16" <your_path>/file.py

```python

@op_all
class Test():

    @op_torch
    @op_automl
    def test_method1():
        pass

    @op_torch
    @op_distributed
    def test_method2():
        pass

     @op_torch
     @op_distributed
     @op_onnxrt16
     def test_method3():
        passs

**Comment by liangs6212** [comment]
http://10.112.231.51:18889/job/BigDL-PRVN-chronos-Python-Spark-3.1-py37-Horovod/827/
http://10.112.231.51:18889/job/BigDL-Chronos-PR-Validation/772/

File: python/chronos/test/bigdl/chronos/autots/model/test_auto_lstm.py [hunk 2]
```diff
@@ -20,10 +20,8 @@
 from unittest import TestCase
 import pytest
 import tempfile
-import onnxruntime
 
-_onnxrt_ver = onnxruntime.__version__ != '1.6.0' #  Jenkins requires 1.6.0(chronos)
-skip_onnxrt = pytest.mark.skipif(_onnxrt_ver, reason="Only runs when onnxrt is 1.6.0")
+from ... import op_all, op_onnxrt16
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/autots/model/test_auto_lstm.py:24]: I'm not quite sure about this syntex: `from ... import `

**Review by TheaperDeng — COMMENTED** [review]
others LGTM

**Comment by liangs6212** [comment]
http://10.112.231.51:18889/job/BigDL-Chronos-PR-Validation/789/

**Pull request merged**